test/playback-watcher.test.js

```diff
@@ -137,6 +166,8 @@ QUnit.test('skips over gap in Chrome due to video underflow', function(assert) {
 
   assert.equal(seeks.length, 1, 'one seek');
   assert.equal(seeks[0], 13, 'player seeked to current time');
+  // ?? not sure if it's correct that there is no skipped gap
+  assert.equal(hlsgapskip, 0, 'there is no skipped gap');
```

Comment:
for this test you'll want to be checking for the `hls-video-underflow` event. The gap skip event wont fire in the video underflow scenario